clang/lib/CIR/CodeGen/CIRGenBuilder.h

```diff
@@ -33,6 +33,15 @@ class CIRGenBuilderTy : public cir::CIRBaseBuilderTy {
       llvm_unreachable("NYI: PPC double-double format for long double");
     llvm_unreachable("Unsupported format for long double");
   }
+
+  bool isSized(mlir::Type ty) {
+    if (mlir::isa<cir::PointerType, cir::ArrayType, cir::BoolType,
+                  cir::IntType>(ty))
+      return true;
+
+    assert(0 && "Unexpected MLIR type");
```

Comment:
I think the 
```suggestion
    assert(!MissingFeatures::unsizedTypes());
```

was the suggestion here (see the other uses of MissingFeatures).

Reply:
Okey, i will check MissingFeatures and update it